UserBaseModel.py

```diff
@@ -0,0 +1,20 @@
+class User:
+  first_name: str
+  last_name: str
+  birthday: str
+  jmbg: str
+  age: int
+  exp: str
+  workHours: int
+  
+  def __init__(self, first_name, last_name, birthday, jmbg, age, exp, workHours):
```

Comment:
Add expected types in the function parameters. 

```suggestion
  def __init__(self, first_name: str, last_name: str, birthday:str, jmbg:str, age:int, exp:str, workHours:str):
```